app/views/edit_phone/resend.html.erb

```diff
@@ -1,21 +1,17 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <%= render "govuk_publishing_components/components/back_link", { href: edit_user_registration_phone_code_path } %>
+<%= render "govuk_publishing_components/components/back_link", { href: edit_user_registration_phone_code_path } %>
 
-    <%= render "govuk_publishing_components/components/heading", {
-      text: t("mfa.phone.resend.heading"),
-      heading_level: 1,
-      font_size: "xl",
-      margin_top: 0,
-      margin_bottom: 3,
-    } %>
+<%= render "govuk_publishing_components/components/heading", {
```

Comment:
Maybe not for this PR, but since every page like this has the same format of heading, I wonder if a further optimisation could be made by declaring the component use at the top level template (or a partial) and passing only the required text to that.

Reply:
Good point, that would reduce repetition even further. There are some differences in heading format between some of the files though (mostly to do with margins it seems?) but maybe we can do some work to unify them anyway.